### ISSUE TITLE
Rails Gem for SurveyJS? Suggestion for community effort

### ISSUE THREAD

**Issue by alexmc6** [surveyjs/survey-library#347]
New Feature / Community Suggestion

Currently SurveyJS only worries about the front end which runs in the browser. It is up to the developer to use either the dxServer, or add their own back end. 

This suggestion is that the SurveyJS community (not necessarily the SurveyJS developers) build a Ruby Gem so that SurveyJS could easily be incorporated into Rails (and Sinatra?) websites. 

The gem would initially fetch the currently tested JQuery version of SurveyJS from github and add it to the asset pipeline. 

It might create a model for storing hand crafted surveys and a model for storing survey responses in a generic way. It might keep track of which surveys a user has done and submitted. 

It might utilise the Rapidfire gem ( https://github.com/code-mancers/rapidfire ) for storing questions and answers, or possibly something else. (Note that Rapidfire has an API under development - we might provide glue which takes the surveyjs submissions and converts them to Rapidfire.

Feel free to comment if you have recommendations for other ways SurveyJS can be incorporated into a Rails website.

**Comment by andrewtelnov** [comment]
@alexmc6 We do not have a lot of expertise on this platform. However, if you have something useful, we definitely add links on the project/repo.

Thank you,
Andrew

**Comment by alexmc6** [comment]
Yeah - I don't think this is your responsibility at all @andrewtelnov . I am just putting a sign post here to see if there is any desire from other SurveyJS users. If nothing else it would encourage me to package up what I am probably writing and release it to others as open source.

**Comment by batmanbury** [comment]
@alexmc6  I'd be very much interested. I'm working on a RoR/jQuery/SurveyJS project now.
